s/icons/bin.svg.ts

```diff
@@ -0,0 +1,6 @@
+import { svg } from "lit"
+
+export default svg `
```

Comment:
we need to sort these icons into subfolders that include the appropriate license files